addons/mail_group/views/mail_group_member_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_group_member_view_list" model="ir.ui.view">
+        <field name="name">mail.group.member.view.list</field>
```

Comment:
Don't we use "tree" ? Even if outdated :D

Reply:
ok :)